build_cpython_linux.rst

```diff
@@ -32,7 +32,7 @@ Get tools and dependencies
         $ sudo dnf install dnf-plugins-core
         $ sudo dnf builddep python3
 
-  * Debian: ``sudo apt-get build-dep python3``
+  * Debian: ``sudo apt-get build-dep -y python3``
```

Comment:
If the -y option is added here, do other parts also need to be added? e.g. `yum`/`dnf` etc.

Reply:
The scope of this PR is just for the Debian system

Reply:
> The scope of this PR is just for the Debian system

OK. And should we open another PR for other system? 

After #16 has been merged, we need resolve conflicting files for this PR. :smile_cat:

Reply:
fixed, thank you. now, if you want to contribute with a PR for Fedora/Redhat, let's go ;-)

Reply:
OK, I will do it. Thanks.